<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageFusion</class>
 <widget class="QDialog" name="imageFusion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1332</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像融合</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="loadImageButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>载入图片</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1311</width>
     <height>481</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="imageListShowlabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>                                                显示图片集</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageFusionShowlabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>                                                 显示合成图像</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>641</width>
     <height>61</height>
    </rect>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="imageFusionButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>融合</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
